### PR TITLE
rapidyaml 0.7.2 (new formula)

### PR DESCRIPTION
**Pull request by cxwx**
new formula rapidyaml 0.7.2

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/r/rapidyaml.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  depends_on "cmake" => :build
+
+  conflicts_with "c4core", because: "both install `c4core`"
+
+  def install
+    system "cmake", ".", "-B", "builddir", "-DCMAKE_INSTALL_PREFIX=#{prefix}", "-DBUILD_SHARED_LIBS=ON"
+    system "cmake", "--build", "builddir"
+    system "cmake", "--install", "builddir"
+  end
```
Comment (chenrui333) [inline Formula/r/rapidyaml.rb:15]: this is more common code style we do in homebrewc-ore

```suggestion
  def install
    system "cmake", "-S", ".", "-B", "build", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
  end

```

**Comment by chenrui333** [comment]
@cxwx why close the pr?

File: Formula/r/rapidyaml.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class Rapidyaml < Formula
+  desc "Library to parse and emit YAML, and do it fast"
+  homepage "https://github.com/biojppm/rapidyaml"
+  url "https://github.com/biojppm/rapidyaml/releases/download/v0.7.2/rapidyaml-0.7.2-src.tgz"
+  sha256 "175b71074005a7d48ae03e973f1a7a5c104c588f193ba85044c5b4a97341aae0"
+
+  depends_on "cmake" => :build
+
+  conflicts_with "c4core", because: "both install `c4core`"
```
Comment (chenrui333) [inline Formula/r/rapidyaml.rb:9]: also I did not see the 4core utility installed.

Comment (cxwx) [reply Formula/r/rapidyaml.rb:9]: The rapidyaml also install a ext `c4core` which conflict with c4core.
like `/opt/homebrew/include/c4` .

**Pull request closed without merging**